client/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "client",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "parcel watch index.html",
+    "build": "npm install --include=dev && parcel build index.html"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@parcel/transformer-react-refresh-wrap": "^2.3.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.2"
+  },
+  "devDependencies": {
+    "parcel": "^2.3.2"
+  }
```

Comment:
Testene kjører ikke fordi dere mangler jest dev dependency i client-prosjektet.

Dere må også ta med `collectCoverageFrom` for å få full uttelling på eksamen. Denne må inkludere *.js, *.jsx, men skal ekskludere genererte filer (`coverage` og `dist`)